opm/material/fluidsystems/blackoilpvt/GasPvtMultiplexer.hpp

```diff
@@ -145,7 +145,7 @@ class GasPvtMultiplexer
                          const Evaluation& temperature,
                          const Evaluation& pressure,
                          const Evaluation& XgO) const
-    { OPM_GAS_PVT_MULTIPLEXER_CALL(return pvtImpl.viscosity(regionIdx, temperature, pressure, XgO)); }
+    { OPM_GAS_PVT_MULTIPLEXER_CALL(return pvtImpl.viscosity(regionIdx, temperature, pressure, XgO)); return 0; }
```

Comment:
I first wanted to go this route too, but the macros can't know the return type of the function -- or even whether there is a return type in the first place.